packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -197,6 +278,47 @@ void main() {
       expect(findStatic(), findsOneWidget);
     });
 
+    testWidgets('CupertinoContextMenu with a builder can change the animation', (WidgetTester tester) async {
+      await tester.pumpWidget(getBuilderContextMenu(builder: (BuildContext context, Animation<double> animation) {
+        return Container(
+          width: 300.0,
+          height: 100.0,
+          decoration: BoxDecoration(
+            color: CupertinoColors.activeOrange,
+            borderRadius: BorderRadius.circular(25.0 * animation.value)
+          ),
+        );
+      }));
+
+      final Widget child = find.descendant(of: find.byType(TickerMode), matching: find.byType(Container)).evaluate().single.widget;
+      final Rect childRect = tester.getRect(find.byWidget(child));
+      expect(find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_DecoyChild'), findsNothing);
+
+      // Start a press on the child.
+      await tester.startGesture(childRect.center);
+      await tester.pump();
+
+      Finder findBuilderDecoyChild() {
+        return find.descendant(
+          of: find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_DecoyChild'),
+          matching: find.byType(Container),
+        );
+      }
+
+      final Container decoyContainer = tester.firstElement(findBuilderDecoyChild()).widget as Container;
+      final BoxDecoration? decoyDecoration = decoyContainer.decoration as BoxDecoration?;
+      expect(decoyDecoration?.borderRadius, equals(BorderRadius.circular(0)));
+
+      expect(find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_DecoyChild'), findsOneWidget);
+      // expect(findBuilderDecoyChild(), findsOneWidget);
```

Comment:
Some commented out code to remove here.  Or actually, why not use findBuilderDecoyChild here?